src/main.rs

```diff
@@ -32,15 +30,17 @@ async fn main() {
     }
 
     let mut output = ServerInfoOutput {
-        servers: HashMap::new(),
-        retry_waits: HashMap::new(),
+        servers: vec![],
+        retry_waits: vec![0; servers.len()],
```

Comment:
Why change from hashmap?

Reply:
there is no viable key for `retry_waits` except if we use indices, and at that point we can just use a vector

Reply:
we aren't using the individual server IPs as keys anymore, and there are no good keys for a `retry_waits` object because all those keys rely on a first response from the server.